src/components/mini-cart/_mini-cart.scss

```diff
@@ -202,6 +202,12 @@
     width: 20px;
     height: 20px;
     margin-right: 8px;
+
+    &.icon--heart {
```

Comment:
What do you think about updating the style name to reflect the state that this edit represents? Not sure what the best term is...unhearted? unliked? We can keep it as-is for now too. Just thought it might be worth considering.

Reply:
You make a good point; this was a miss on my end. I've updated it to heart-outline.